docker-compose.yml

```diff
@@ -1,8 +1,6 @@
-version: "3"
-
 services:
   selenium:
-    image: ${SELENIUM_IMAGE:-selenium/standalone-chrome:4}
+    image: ${SELENIUM_IMAGE:-selenium/standalone-chromium:4}
```

Comment:
The main reason `chrome` was used by default is mainly the [high market share](https://gs.statcounter.com/browser-market-share) which also means many users would expect such a browser by default (we had `firefox` before). That said, it has been defined as a variable so the user can always use a different image entirely (even a non-selenium/standalone image). Therefore I would recommend removing this change.